src/dockerfiles/aqueduct.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9
+
+MAINTAINER Aqueduct <hello@aqueducthq.com> version: 0.0.1
+
+USER root
+
+ENV PYTHONUNBUFFERED 1
+
+RUN pip install aqueduct-ml==0.0.16
```

Comment:
nit: should the version be passed in as an arg?

Reply:
I think we will version this image with the release.

Reply:
Added the version as a variable.